packages/divider-curve/src/DividerCurve.tsx

```diff
@@ -0,0 +1,20 @@
+import type React from 'react'
+
+import DividerCurveOne from './assets/divider-curve-variation-one'
+import DividerCurveTwo from './assets/divider-curve-variation-two'
+
+export interface ComponentProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * The curver divider variant, defaults to 1
+   */
+  variant?: number
```

Comment:
Variant 1 and 2 is not too descriptive of a prop. How does a dev using this component what the difference is?

Reply:
I could describe them visually as something like "wavyLine" & "invertedCurve", or simply "top" and "bottom" as they are always in that order.

While the first line is always associated with the Education section, the second line is used in different sections, so not sure I can use section as context.

Reply:
I think those props much better. We can make them bool props

Reply:
Updated to bool `inverted` prop